gsa/src/gmp/parser.js

```diff
@@ -94,7 +95,7 @@ export const YES_VALUE = 1;
 export const NO_VALUE = 0;
 
 export const parseYesNo = value =>
-  value === '1' || value === 1 ? YES_VALUE : NO_VALUE;
+  value === '1' || value === 1 || value === true ? YES_VALUE : NO_VALUE;
```

Comment:
This function should not be changed. It may have serious side effects.
```suggestion
  value === '1' || value === 1 ? YES_VALUE : NO_VALUE;
```